src/encoded/tests/features/matrix_reference_epigenome.featurematrix.feature

```diff
@@ -0,0 +1,29 @@
+@matrix @usefixtures(workbook)
```

Comment:
very confused by the presence of this file in the PR...

Reply:
This was a weird pytest issue where the file as 
src/encoded/tests/features/matrix_reference_epigenome.feature 

was being looked for at 

src/encoded/tests/features/matrix_reference_epigenome.featurematrix.feature

Also, if you look at the commit history you'll see this as flag with WARNING.